docker/Dockerfile

```diff
@@ -38,14 +37,7 @@ RUN groupadd --gid $groupid app && \
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     apt-transport-https build-essential curl git libpq-dev \
-    gettext libffi-dev jed
-
-# Install dump_syms
-RUN DEBIAN_FRONTEND=noninteractive \
-    apt-get install -y --no-install-recommends \
-    gyp ninja-build binutils-gold gcc-6 g++-6 pkg-config cabextract
-COPY ./docker/build_dump_syms.sh /tmp
-RUN /tmp/build_dump_syms.sh
```

Comment:
This builds `dump_syms`. I removed this step from the Dockerfile as well as the `build_dump_syms.sh` script.